src/components/carousel/ThumbnailNavigationButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+import { Direction } from './NavigationButton';
+import { ChevronLeftSmallIcon, ChevronRightSmallIcon } from '../icons';
+import Flex from '../flex';
+
+interface Props {
+  onClick: () => void;
+  direction: Direction;
+}
+
+const icons = {
+  previous: <ChevronLeftSmallIcon />,
+  next: <ChevronRightSmallIcon />,
+};
+
+const ThumbnailNavigationButton: FC<Props> = ({ direction, onClick }) => {
+  const { paginationButtonContainer, paginationButton } = useMultiStyleConfig(
+    'Carousel',
+    {
+      variant: 'fullScreen',
+    }
+  );
+  const side = direction === 'previous' ? { left: '0' } : { right: '0' };
+
+  return (
+    <chakra.button
```

Comment:
Are we supposed to use the charka components directly for cases like this?

Reply:
I think it's fine in this case. This is a very specific component that should work well on dark background

Reply:
I would say so - the `chakra` wrapper allows us to leverage chakra-specific props like `__css` or `sx`. At the same time this is so much different from the `Button` component that it doesn't make sense to use it